upup/pkg/fi/cloudup/awsup/machine_types.go

```diff
@@ -1697,7 +1587,7 @@ var MachineTypes []AWSMachineTypeInfo = []AWSMachineTypeInfo{
 	// r3 family
 	{
 		Name:              "r3.large",
-		MemoryGB:          15.25,
+		MemoryGB:          15,
```

Comment:
The table [here](https://aws.amazon.com/ec2/previous-generation/) gives the 15.25 number.